include/boost/geometry/algorithms/detail/overlay/handle_colocations.hpp

```diff
@@ -475,7 +489,8 @@ inline void gather_cluster_properties(Clusters& clusters, Turns& turns,
             turn_operation_type& op = turn.operations[ranked.operation_index];
 
             if (set_startable
-                    && for_operation == operation_union && cinfo.open_count == 0)
+                && for_operation == operation_union
```

Comment:
Here is the condition for `if constexpr`. Or is it not always the case that `for_operation` is known at compile-time?

Reply:
Yes, it is known. I'll update.

Reply:
I didn't update this one, and the ones below.
Because `for_operation` is passed as a normal parameter.
This needs more refactoring and it is better to separate that.